src/delete_atoms.cpp

```diff
@@ -949,19 +936,17 @@ void DeleteAtoms::molring(int n, char *cbuf, void *ptr)
   auto *daptr = (DeleteAtoms *) ptr;
   auto *list = (tagint *) cbuf;
   int *dlist = daptr->dlist;
-  std::map<tagint, int> *hash = daptr->hash;
   int nlocal = daptr->atom->nlocal;
   tagint *molecule = daptr->atom->molecule;
 
   // cbuf = list of N molecule IDs from other proc, put them in hash
 
-  hash->clear();
-  for (int i = 0; i < n; i++) (*hash)[list[i]] = 1;
+  std::set<tagint> hash(list, list + n);
```

Comment:
I'm curious, is there an advantage to set over unordered_set here?

Reply:
Probably not.